examples/sampling_c3/joint_trajectory_generator.cc

```diff
@@ -35,10 +34,18 @@ JointTrajectoryGenerator::JointTrajectoryGenerator(
       plant.num_velocities(), plant.num_actuators()))
     .get_index();
 
+  target_joint_position_port_ =
+    this->DeclareVectorInputPort("lcmt_franka_target_joint_position",
```

Comment:
A bit misleading to have `lcmt_` in the name of the input port when the value is a vector input, instead of the lcmt type.